change/@ni-nimble-angular-473e22fd-968c-4658-8f4c-c43d789a4054.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The dialogs in the Angular and Blazor example apps are taller than they need to be and the title doesn't appear at the top.
![image](https://user-images.githubusercontent.com/10500124/193084866-76000c32-0753-4f7f-a2d1-96a56a384efa.png)

I'm assuming this is because of changes to slots. If you can capture what clients need to change succinctly it would be good to add it to the change file comments.

Reply:
That's a good point.

I'll also update our example apps to use the slots, since that's what matches our designs.